pkg/neorpc/rpcevent/filter_test.go

```diff
@@ -117,7 +117,7 @@ func TestMatches(t *testing.T) {
 			name: "block, primary mismatch",
 			comparator: testComparator{
 				id:     neorpc.BlockEventID,
-				filter: neorpc.BlockFilter{Primary: &badPrimary},
+				filter: neorpc.SubscriptionFilter(&neorpc.BlockFilter{Primary: &badPrimary}),
```

Comment:
Useless change. If IsValid is defined not on the pointer, then it should be OK without `neorpc.SubscriptionFilter` assertion.